Comp2023/src/main/java/frc/robot/commands/AutoPreloadHigh.java

```diff
@@ -42,14 +43,16 @@ public AutoPreloadHigh(Elbow elbow, Extension extension, Wrist wrist, Gripper gr
         new PrintCommand(getName() + ": Move Extension for Preload"),   
         new ExtensionMoveToLength(extension, ExtensionLength.EXTENSION_HIGH),
 
-        new PrintCommand(getName() + ": Move Wrist for Preload"),   
-        new WristMoveToAngle(wrist, WristAngle.WRIST_SCORE),
+        new SequentialCommandGroup(

```

Comment:
These commands are already sequential (this command is a sub-classed from the SequentialCommandGroup class), so we don't need to wrap the following commands within another SequentialCommandGroup.